_locales/_untranslated_en.json

```diff
@@ -43,5 +43,8 @@
   },
   "copy_json": {
     "message": "Copy JSON"
+  },
+  "copy_text": {
```

Comment:
maybe just use the word "Copy" for the default (obvious) action or if there is only one copy action.

"Copy link" etc. can be used for the more specialized ones and if there is more than one action.